src/reactpy/_html.py

```diff
@@ -104,7 +104,7 @@ def _fragment(
     event_handlers: EventHandlerDict,
 ) -> VdomDict:
     """An HTML fragment - this element will not appear in the DOM"""
-    if attributes or event_handlers:
+    if (attributes and any(a != "key" for a in attributes)) or event_handlers:
```

Comment:
Is this change required? Using `any` here would be pretty expensive for large attribute dictionaries.